cypress/integration/addNewClient/client.js

```diff
@@ -15,21 +15,25 @@ describe("Test for adding new client ", () => {
   });
 
   it("should verify validation error message while submitting without mandatory fields", () => {
-    const selector = [".js-input-error", ".js-input-error", ".js-input-error"];
     const errorMessage = [
```

Comment:
Since this array contains empty validation error message .You can use variable naming as emptyValidationError